chainer/testing/function_link.py

```diff
@@ -70,16 +70,22 @@ def __init__(self, argument_value, expected_initializer):
 
 
 class FunctionTestBase(object):
+
     backend_config = None
-    check_forward_options = {}  # type: tp.Dict[str, tp.Any]
-    check_backward_options = {}  # type: tp.Dict[str, tp.Any]
-    check_double_backward_options = {}  # type: tp.Dict[str, tp.Any]
+    check_forward_options = None
+    check_backward_options = None
+    check_double_backward_options = None
     skip_forward_test = False
     skip_backward_test = False
     skip_double_backward_test = False
     dodge_nondifferentiable = False
     contiguous = None
 
+    def __init__(self):
```

Comment:
Could you try the following to fix the test failure?
```suggestion
    def __init__(self, *args, **kwargs):
        super(FunctionTestBase, self).__init__(*args, **kwargs)
```

Reply:
Yes, will change.

Reply:
And thanks for the suggestion!

Reply:
For the record, `super` here follows the MRO (`FunctionTestCase`, `FunctionTestBase`, `unittest.TestCase`) and will refer to `unittest.TestCase`.